tests/test_forgot_password.py

```diff
@@ -2,9 +2,6 @@
 import pytest
 from assertpy import assert_that
 
-from pages.about_page import AboutPage
-from pages.forgot_password_page import ForgotPasswordPage
-from pages.login_page import LoginPage
 from utils.excel_parser import ExcelParser
 
 
```

Comment:
You can use : @pytest.mark.usefixtures ("fixture1","fixture2",....)
On class level instead of sending the fixtures each time as arguments.
that way, the test methods will have access to these fixtures.